### PR TITLE
Add network status

### PR DESCRIPTION
**Pull request by miladsoft**

### CODE REVIEW

File: src/Blockcore.AtomicSwaps.Client/SwapsConfiguration.cs [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
-            new IndexerUrl { Symbol = "IMPLX", Url = "https://implx.indexer.blockcore.net/api" },
-            new IndexerUrl { Symbol = "RSC", Url = "https://rsc.indexer.blockcore.net/api" },
-            new IndexerUrl { Symbol = "SBC", Url = "https://sbc.indexer.blockcore.net/api" },
-        };
+            return _storage.GetExplorerUrl();
+        }
+        public List<IndexerUrl> Indexers()
+        {
+            List<IndexerUrl> indexers = new List<IndexerUrl>();
+            indexers.Add(new IndexerUrl { Symbol = "BTC", Url = _storage.GetIndexerUrl("BTC") ?? "https://btc.indexer.coinvault.io/api" });
```
Comment (sondreb) [inline src/Blockcore.AtomicSwaps.Client/SwapsConfiguration.cs:30]: All of these URLs should be retrieved from the dynamic DNS setup, shouldn't it?

**Review by sondreb — APPROVED** [review]

File: src/Blockcore.AtomicSwaps.Client/Storage.cs [hunk 2]
```diff
@@ -11,6 +11,11 @@ public Storage(ISyncLocalStorageService storage)
         {
             _storage = storage;
         }
+
+        public Storage()
```
Comment (dangershony) [inline src/Blockcore.AtomicSwaps.Client/Storage.cs:15]: if this is used `_storage` will fail when called

File: src/Blockcore.AtomicSwaps.Client/SwapsConfiguration.cs (hunk 1 above)
Comment (dangershony) [reply src/Blockcore.AtomicSwaps.Client/SwapsConfiguration.cs:30]: shall I wait for this to be pushed?

**Review by dangershony — APPROVED** [review]

**Comment by dangershony** [comment]
Any changes just do them in a new PR

**Pull request merged**